ocs_ci/deployment/aws.py

```diff
@@ -63,6 +63,9 @@ def __init__(self):
         # dict of cluster prefixes with special handling rules (for existence
         # check or during a cluster cleanup)
         self.cluster_prefixes_special_rules = CLUSTER_PREFIXES_SPECIAL_RULES
+        ocs_version = version.get_semantic_ocs_version_from_config()
```

Comment:
since gp2-csi chnages are from OCP side, we can set DEFAULT_STORAGECLASS based on running OCP version

Reply:
makes sense to change it to OCP version, but not running one as this is the deployment class and we don't have the OCP running yet.